docs/hooks/hooks.md

```diff
@@ -6,4 +6,1335 @@ sidebar_position: 4
 
 Scaffold-ETH 2 provides a collection of custom React hooks designed to simplify interactions with your deployed smart contracts. These hooks are wrappers around `wagmi`, automatically loading the necessary contract ABI and address. These hooks offer a simplified and streamlined interface for interacting with your smart contracts.
 
-If you need to interact with external contracts, you can use `wagmi` directly, or add external contract data to your `deployedContracts.ts` file.
+If you need to interact with external contracts you can use `wagmi` directly, or you can add external contract data to your `externalContracts.ts` file, which would let you use Scaffold-ETH 2 hooks with autocompletion to interact with them.
```

Comment:
My suggestion:

"If you need to interact with external contracts (i.e. not deployed with your SE-2 instance) you can add external contract data to your `packages/nextjs/contracts/externalContracts.ts` file, which would let you use Scaffold-ETH 2 hooks"